src/app/pages/ParaTimesPage/TransactionAmount/index.tsx

```diff
@@ -124,7 +138,9 @@ export const TransactionAmount = () => {
                   }}
                   plain
                   label="MAX"
-                  onClick={() => setTransactionForm({ ...transactionForm, amount: formatter(balance) })}
+                  onClick={() =>
```

Comment:
MAX doesn't work with fees.

E.g. withdrawing MAX from cipher (after removing setAllowance) fails with `Unknown gRPC error: withdraw: insufficient runtime balance`

Reply:
I will address this one in a new PR.